service/src/ConstConfigIOGroup.cpp

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "ConstConfigIOGroup.hpp"
+
+#include <cmath>
+
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+#include "geopm/Helper.hpp"
+#include "geopm/PlatformTopo.hpp"
+#include "geopm/Exception.hpp"
+#include "geopm/Agg.hpp"
+#include "geopm_topo.h"
+#include "config.h"
```

Comment:
```suggestion
#include "config.h"

#include "ConstConfigIOGroup.hpp"

#include <cmath>
#include <iostream>
#include <iterator>
#include <sstream>

#include "geopm_topo.h"
#include "geopm/Helper.hpp"
#include "geopm/PlatformTopo.hpp"
#include "geopm/Exception.hpp"
#include "geopm/Agg.hpp"
```

Reply:
Is Helper.hpp needed for anything?  I thought read_file() would be called somewhere, but I don't see it.